test/boost/cql_query_test.cc

```diff
@@ -3189,13 +3189,21 @@ SEASTAR_TEST_CASE(test_static_multi_cell_static_lists_with_ckey) {
             auto msg = e.execute_cql("SELECT slist, v FROM t WHERE p = 1 AND c = 1;").get0();
             auto slist_type = list_type_impl::get_instance(int32_type, true);
             assert_that(msg).is_rows().with_row({
-                { slist_type->decompose(make_list_value(slist_type, list_type_impl::native_type({3}))) },
+                { slist_type->decompose(make_list_value(slist_type, list_type_impl::native_type({data_value(3)}))) },
```

Comment:
Looks like an unrelated change

Reply:
yes, it's necessary, but should have been in its own patch. I'll split it off.